sdcm/sct_config.py

```diff
@@ -1439,6 +1439,9 @@ class SCTConfiguration(dict):
              help="The time interval in minutes which gets waited before the KMS key rotation happens."
                   " Applied when the AWS KMS service is configured to be used."),
 
+        dict(name="enable_azure_key_rotation", env="SCT_ENABLE_AZURE_KEY_ROTATION", type=boolean,
```

Comment:
what is the option we have for AWS for this?

Reply:
AWS does rotation for all jobs by default. Here we need to restrict Azure key rotation to specific jobs.

Reply:
Updated PR.